lib/vmdb/fast_gettext_helper.rb

```diff
@@ -1,23 +1,14 @@
 module Vmdb
   module FastGettextHelper
-    def self.human_locale(locale)
-      # TRANSLATORS: Provide locale name in native language (e.g. English, Deutsch or Português)
-      human_locale = _("locale_name")
-      human_locale = locale if human_locale == "locale_name"
-      human_locale
-    end
-
     def self.register_human_localenames
-      original_locale = FastGettext.locale
+      human_locale_names = YAML.load_file(Rails.root.join('config/human_locale_names.yaml'))['human_locale_names']
```

Comment:
What we're doing during application start is:
1. First we're registering locales -- either by looking what's on the filesystem or by looking into a yaml file
containing list of supported locales.
2. Next, for every locale registered in previous step, we're registering a nice locale name to be displayed
in the UI.

So I'd say that `self.register_human_localenames()` is by definition already aware of supported locales,
by definition it does not care about anything else but locales that were registered as supported.